spec/request/letters_request_spec.rb

```diff
@@ -139,13 +127,73 @@
       it 'should return a not found response' do
         VCR.use_cassette('evss/letters/beneficiary_500') do
           get '/v0/letters/beneficiary', nil, auth_header
-          expect(response).to have_http_status(:ok)
-          expect(response).to match_response_schema('letter_beneficiary')
-          expect(Oj.load(response.body).dig('meta', 'status')).to eq(
-            Common::Client::ServiceStatus::RESPONSE_STATUS[:server_error]
-          )
+          expect(response).to have_http_status(:internal_server_error)
+          expect(response).to match_response_schema('letters_errors')
         end
       end
     end
   end
+
+  describe 'error handling' do
+    context 'with a letter generator service error' do
+      it 'should return a not found response' do
+        VCR.use_cassette('evss/letters/letters_letter_generator_service_error') do
```

Comment:
Are you generating these VCR's manually, one by one? Ideally, a developer would be able to just delete the cassettes and re-run the entire spec to re-generate with the one caveat being that they will need to be connected to EVSS openvpn.  Otherwise, it's a real pain to regenerate these.